ecs-configuration/elasticsearch/ecs-fields-tls.json

```diff
@@ -37,8 +37,16 @@
                 "type": "keyword"
               },
               "ja3": {
-                "ignore_above": 1024,
-                "type": "keyword"
+                "properties": {
+                  "hash": { 
+                    "ignore_above": 8000,
+                    "type": "keyword"
+                  },
+                  "string" : {
```

Comment:
can we split this `ja3.string` out to a new field called `ja3_string`? The reason being that `tls.ja3` is an ECS schema field, so converting it to an object is a breaking change with that config.

Reply:
Thats fine, the default for suricata eve.json breaks it into an object so I just followed that so that Zeek would not conflict because we can not have `tls.ja3` be both a field and an object.